build.sh

```diff
@@ -5,11 +5,24 @@ version=$1
 commit=${TRAVIS_COMMIT:-`git rev-parse --verify HEAD`}
 echo "Version: ${version:=`date -u +local-%Y%m%d-%H%M%S`}"
 
-for os in linux darwin windows; do
+
+for os in linux; do  # add darwin and windows in the future
     export GOOS=$os
+    go clean
+    go install github.com/Azure/adx-automation-agent/common
+    go install github.com/Azure/adx-automation-agent/models
+    go install github.com/Azure/adx-automation-agent/httputils
+    go install github.com/Azure/adx-automation-agent/kubeutils
```

Comment:
Do you truly need all this binaries?

Reply:
Yes. The definitions are spreading out.

Reply:
But just building the droid and the dispatcher should also take the code in this packages. Building all imported packages is not needed.

Reply:
Oh. I see.